src/skeletons/SkeletonTable.js

```diff
@@ -8,55 +8,47 @@ import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTabl
 
 import './SkeletonTable.scss';
 
-class SkeletonTable extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      columns: [
-        { title: <Skeleton size="xs" /> },
-        { title: <Skeleton size="sm" /> },
-        { title: <Skeleton size="sm" /> },
-        { title: <Skeleton size="md" /> },
-        { title: <Skeleton size="sm" /> },
-      ],
-      rows: [
-        [
-          { title: <Skeleton size="md" /> },
-          { title: <Skeleton size="xs" /> },
-          { title: <Skeleton size="xs" /> },
-          { title: <Skeleton size="md" /> },
-          { title: <Skeleton size="md" /> },
-        ],
-      ],
-    };
-  }
+const SkeletonTable = ({ ...props }) => {
```

Comment:
Could we maybe use the `SkeletonTable` component from the component-groups package and just drop this component?

Reply:
You are right, I removed the `SkeletonTable` refactor from this PR and I will create another PR where I delete the `SkeletonTable`